net-p2p/bitcoind/bitcoind-0.11.1.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 2010-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+BITCOINCORE_COMMITHASH="cf33f196e79b1e61d6266f8e5190a0c4bfae7224"
+BITCOINCORE_LJR_DATE="20150921"
+BITCOINCORE_IUSE="addrindex examples ljr logrotate test upnp +wallet zeromq"
+BITCOINCORE_ADDRINDEX_DIFF="8a915e56f4be5d090d79f01cbd4a23ce1b7e9168...4a6331c5068d8f2c95731518445a57267d506bb5"
+BITCOINCORE_ADDRINDEX_PATCHFILE="bitcoin-addrindex-v0.11.0.patch"
+BITCOINCORE_POLICY_PATCHES="cltv cpfp dcmp rbf spamfilter"
+BITCOINCORE_NEED_LEVELDB=1
+BITCOINCORE_NEED_LIBSECP256K1=1
+inherit bash-completion-r1 bitcoincore user systemd
+
+DESCRIPTION="Original Bitcoin crypto-currency wallet for automated services"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="
+	logrotate? (
+		app-admin/logrotate
+	)
+"
+DEPEND="${RDEPEND}"
+
+pkg_setup() {
+	local UG='bitcoin'
+	enewgroup "${UG}"
+	enewuser "${UG}" -1 -1 /var/lib/bitcoin "${UG}"
+}
+
+src_prepare() {
+	sed -i 's/have bitcoind &&//;s/^\(complete -F _bitcoind bitcoind\) bitcoin-cli$/\1/' contrib/${PN}.bash-completion || die
+	bitcoincore_src_prepare
+}
+
+src_configure() {
+	bitcoincore_conf \
+		--with-daemon
+}
+
+src_install() {
+	bitcoincore_src_install
+
+	insinto /etc/bitcoin
+	newins "${FILESDIR}/bitcoin.conf" bitcoin.conf
+	fowners bitcoin:bitcoin /etc/bitcoin/bitcoin.conf
+	fperms 600 /etc/bitcoin/bitcoin.conf
+
+	newconfd "contrib/init/bitcoind.openrcconf" ${PN}
+	newinitd "contrib/init/bitcoind.openrc" ${PN}
+	systemd_dounit "${FILESDIR}/bitcoind.service"
+
+	keepdir /var/lib/bitcoin/.bitcoin
+	fperms 700 /var/lib/bitcoin
+	fowners bitcoin:bitcoin /var/lib/bitcoin/
+	fowners bitcoin:bitcoin /var/lib/bitcoin/.bitcoin
+	dosym /etc/bitcoin/bitcoin.conf /var/lib/bitcoin/.bitcoin/bitcoin.conf
+
+	dodoc doc/assets-attribution.md doc/bips.md doc/tor.md
+	doman contrib/debian/manpages/{bitcoind.1,bitcoin.conf.5}
+
+	newbashcomp contrib/${PN}.bash-completion ${PN}
+
+	if use examples; then
+		docinto examples
+		dodoc -r contrib/{bitrpc,qos,spendfrom,tidy_datadir.sh}
+	fi
+
+	if use logrotate; then
```

Comment:
We don't want it in RDEPEND either, since the only reason it is there is a single file. We don't do that. If people use logrotate, they install it directly and then can use the files that are unconditionall installed by this ebuild. We do the same with systemd. We do NOT want logrotate USE flags just for a single file.